src/query/sql/src/planner/binder/bind_table_reference/bind_table.rs

```diff
@@ -130,16 +131,24 @@ impl Binder {
                 }
                 if e.code() == ErrorCode::UNKNOWN_DATABASE {
                     return Err(ErrorCode::UnknownDatabase(format!(
-                        "Unknown database `{}` in catalog '{catalog}'",
-                        database
+                        "Unknown database `{database}` in catalog '{catalog}'",
                     ))
                     .set_span(*span));
                 }
                 if e.code() == ErrorCode::UNKNOWN_TABLE {
-                    return Err(ErrorCode::UnknownTable(format!(
-                        "Unknown table `{database}`.`{table_name}` in catalog '{catalog}'"
-                    ))
-                    .set_span(*span));
+                    let name = &table.name;
+                    let err = match self.name_resolution_ctx.not_found_suggest(table) {
```

Comment:
This will more short:
```
    let err_message = match self.name_resolution_ctx.not_found_suggest(table) {
        NameResolutionSuggest::Quoted => format!(
            "Unknown table `{database}`.{name} (unquoted) in catalog '{catalog}'. Did you mean `{name}` (quoted)?"
        ),
        NameResolutionSuggest::Unqoted => format!(
            "Unknown table `{database}`.`{name}` (quoted) in catalog '{catalog}'. Did you mean {name} (unquoted)?"
        ),
        NameResolutionSuggest::None => format!(
            "Unknown table `{database}`.`{name}` in catalog '{catalog}'"
        ),
    };
    
    ... ...
```